libpod/stats.go

```diff
@@ -34,6 +36,14 @@ func (c *Container) GetContainerStats(previousStats *ContainerStats) (*Container
 		return stats, define.ErrCtrStateInvalid
 	}
 
+	if previousStats == nil {
```

Comment:
You need to make sure all `GetContainerStats()` callers (see cmd/podman/stats.go) pass in `nil` instead of `&libpod.ContainerStats{}`

Reply:
I've updated all that I saw, and still, the segv.  I also changed some code in varlink to account for this, and I was a bit leery about that.